tests/unit/colorpicker/tests.html

```diff
@@ -43,54 +45,61 @@
 
-	        deepEqual($(this.actualColorPicker._colorTable).html(), expectedHtmlMarkup, "_create does not append right markup");
+	        deepEqual($(this.actualColorPicker.igColorPicker("colorTable")).html(), expectedHtmlMarkup, "color table does not append right markup.");
 	    });
 
 	    test("_changeSelectedColor", function () {
-	        this.actualColorPicker._changeSelectedColor(this.firstColor);
-	        ok(this.firstColor.hasClass("selected-color"), "first color wasn't selected");
+	        this.actualColorPicker.igColorPicker("selectColor", this.firstColor);
+	        ok(this.firstColorElement.hasClass("selected-color"), "first color wasn't selected.");
 	    });
 
 	    test("_changeColorOptions", function() {
-	    	this.actualColorPicker._setOption("colors", []);
-	    	equal(this.actualColorPicker.options.colors.length , 0, "colors options was not set to empty through _setOption method");
+	    	this.actualColorPicker.igColorPicker("option", "colors", []);
+	    	equal(this.actualColorPicker.igColorPicker("customColorTable").children().length, 0, "colors were not removed as elements.")
 	    })
 	    
 	    test("_changeStandardColorsOption", function () {
 	    	var colors = ["#f0ff0f", "#000a00", "#FEECE1"];
 	    	
-	    	notEqual(this.actualColorPicker.options.standardColors, colors, "standardColors options is equal with givan colors");
+	    	notEqual(this.actualColorPicker.igColorPicker("option", "standardColors"), colors, "standardColors options is equal with givan colors.");
 	    	
-	    	this.actualColorPicker._setOption("standardColors", colors)
-	    	equal(this.actualColorPicker.options.standardColors, colors, "array of standardColors option was not changed through _setOption method");
+	    	this.actualColorPicker.igColorPicker("option", "standardColors", colors)
+	    	deepEqual(this.actualColorPicker.igColorPicker("option", "standardColors"), colors, "array of standardColors option was not changed."); // deepEqual
+
+	    	equal(this.actualColorPicker.igColorPicker("option", "standardColors").length, colors.length, "standard colors length isn't equal with given colors length.");
```

Comment:
Here you should include the same test as above, where you retrieve the standardColorsTable and check the length of the children. This will require one more API method, I guess :)